test/aphroditeInterfaceFactory_test.js

```diff
@@ -1,17 +1,23 @@
 import { expect } from 'chai';
-import * as aphrodite from 'aphrodite';
+import sinon from 'sinon-sandbox';
+import aphrodite from 'aphrodite';
 import aphroditeInterfaceFactory from '../src/aphroditeInterfaceFactory';
 
+import * as resolve from '../src/utils/resolve';
+
 describe('aphroditeInterfaceFactory', () => {
   const { StyleSheetTestUtils } = aphrodite;
   const aphroditeInterface = aphroditeInterfaceFactory(aphrodite);
+  let resolveSpy;
 
   beforeEach(() => {
     StyleSheetTestUtils.suppressStyleInjection();
+    resolveSpy = sinon.spy(resolve, 'default');
```

Comment:
this approach is very brittle, and will break when babel starts working like real modules - the same goes for any place we're using `import *` :-/

Reply:
I will fix it then when babel starts working like real modules... there's literally no other way to test this and it seems like a valuable test to me.